tests/e2e/e2e_test.go

```diff
@@ -47,10 +47,6 @@ var _ = BeforeSuite(func() {
 	curProj = runCmd("oc project -q")
 })
 
-var _ = AfterSuite(func() {
```

Comment:
Why are these lines removed?

Reply:
after suite wont work in the logout state, this function doing nothing other than `oc project list` as you can see

Reply:
ok